queue-manager/rango-preset/package.json

```diff
@@ -35,7 +34,7 @@
     "@types/uuid": "^8.3.4"
   },
   "dependencies": {
-    "@sentry/browser": "^6.12.0",
+    "@rango-dev/logging-core": "0.1.0",
```

Comment:
Is it intentional that we used fix version here?

Reply:
Nice catch! missed this one.